plugins/base/src/test/kotlin/enums/JavaEnumsTest.kt

```diff
@@ -57,4 +68,50 @@ class JavaEnumsTest : BaseAbstractTest() {
             }
         }
     }
+
+    @Test
+    fun `should mark synthetic functions generated for Kotlin as obvious`() {
```

Comment:
Looks like some classes got duplicated for some reason, probably due to automatic conflict resolution. Deleted `JavaEnumTest` and moved this test to `JavaEnumsTest`, that was the only difference